### DISCUSSION TITLE
How to assign keyword or label to several selected photos

### DISCUSSION THREAD

**Discussion by DarkDau** [photoprism/photoprism#3899] [Q&A]
How can I assign a keyword or label to several selected photos in photoprism?
I can only edit the images one after the other via review, not all at once. 
For example, I would like to assign a country or a city to several photos in edit mode.
Or a name to the pet as a label

**Comment by graciousgrey** [comment]
Batch editing is planned but not implemented yet, see

- #271

**Reply by DarkDau** [reply]
thank you. 
This means we can expect bulk edit in the near future?!

**Reply by graciousgrey** [reply]
We do our best to get to it as soon as possible however we need to finish work on some other tasks first.

**Reply by DarkDau** [reply]
Don't worry, I'll wait.
...and certainly donate something or become a member. Your Photoprism is a really nice project!
In the meantime I will use https://github.com/Blendan1/PhotoPrismBulkEditor
